src/redux/editOptions.js

```diff
@@ -1,262 +1,28 @@
-import analytics from '@segment/analytics-react-native';
-import produce from 'immer';
-import { concat, difference, filter, union, uniq, without } from 'lodash';
-import { Value } from 'react-native-reanimated';
-import {
-  getHiddenCoins,
-  getPinnedCoins,
-  saveHiddenCoins,
-  savePinnedCoins,
-} from '../handlers/localstorage/accountLocal';
-import EditOptions from '../helpers/editOptionTypes';
+// TODO get rid of this redux state soon. Currently we use this in a few selectors
```

Comment:
as `hiddenCoins` is also part of the mmkv migration, not sure why this redux is still needed here.
Only place that still uses the `setHiddenCoins` from this file is `redux/data.js` which also updates the MMKV

Reply:
we still use this in one selector so we cannot remove this. I defer to the redux migration work.